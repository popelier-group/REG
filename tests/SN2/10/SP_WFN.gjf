# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.65254800   -0.00006000    0.00021200
 H                 -0.84244400   -0.89873400   -0.55974900
 H                 -0.84246000    0.93422900   -0.49804500
 H                 -0.84234800   -0.03569000    1.05848900
 Br                 1.59839800    0.00000300   -0.00001200
 Cl                -3.31052600    0.00001900   -0.00006500

frame.wfn
