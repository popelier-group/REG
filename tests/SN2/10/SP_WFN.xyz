
6

C        0.000100   -0.007805    0.541384
H       -0.572694   -0.901002    0.718413
H       -0.484780    0.930703    0.744648
H        1.057723   -0.061280    0.730642
Br        0.000309    0.024457   -1.709330
Cl       -0.000687   -0.045740    3.199092
