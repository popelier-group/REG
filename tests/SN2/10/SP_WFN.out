 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.10/Gau-18028.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.10/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     18029.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.65255  -0.00006   0.00021 
 H                    -0.84244  -0.89873  -0.55975 
 H                    -0.84246   0.93423  -0.49805 
 H                    -0.84235  -0.03569   1.05849 
 Br                    1.5984    0.       -0.00001 
 Cl                   -3.31053   0.00002  -0.00007 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.652548   -0.000060    0.000212
      2          1           0       -0.842444   -0.898734   -0.559749
      3          1           0       -0.842460    0.934229   -0.498045
      4          1           0       -0.842348   -0.035690    1.058489
      5         35           0        1.598398    0.000003   -0.000012
      6         17           0       -3.310526    0.000019   -0.000065
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.075747   0.000000
     3  H    1.075743   1.834001   0.000000
     4  H    1.075753   1.833995   1.833996   0.000000
     5  Br   2.250946   2.660591   2.660564   2.660628   0.000000
     6  Cl   2.657978   2.685597   2.685532   2.685836   4.908924
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000100   -0.007805    0.541384
      2          1           0       -0.572694   -0.901002    0.718413
      3          1           0       -0.484780    0.930703    0.744648
      4          1           0        1.057723   -0.061280    0.730642
      5         35           0        0.000309    0.024457   -1.709330
      6         17           0       -0.000687   -0.045740    3.199092
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         149.0851703           0.8510629           0.8510628
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       174.4696937377 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  8.01D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.39011058     A.U. after   11 cycles
            NFock= 11  Conv=0.95D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.65513-101.28238 -62.30805 -56.12750 -56.12495
 Alpha  occ. eigenvalues --  -56.12495 -10.08492  -9.19574  -8.51701  -6.95525
 Alpha  occ. eigenvalues --   -6.95335  -6.95335  -6.34531  -6.33701  -6.33701
 Alpha  occ. eigenvalues --   -2.45665  -2.45437  -2.45437  -2.44780  -2.44780
 Alpha  occ. eigenvalues --   -0.61724  -0.55183  -0.52095  -0.30201  -0.30201
 Alpha  occ. eigenvalues --   -0.19821  -0.13338  -0.13338  -0.07948  -0.07935
 Alpha  occ. eigenvalues --   -0.07935
 Alpha virt. eigenvalues --    0.09420   0.11297   0.14751   0.14751   0.15557
 Alpha virt. eigenvalues --    0.17768   0.18198   0.18198   0.20732   0.21907
 Alpha virt. eigenvalues --    0.21907   0.23781   0.23781   0.24801   0.31236
 Alpha virt. eigenvalues --    0.31659   0.31660   0.35545   0.39775   0.51645
 Alpha virt. eigenvalues --    0.53838   0.53838   0.55474   0.55474   0.63480
 Alpha virt. eigenvalues --    0.66132   0.66133   0.70304   0.75232   0.75748
 Alpha virt. eigenvalues --    0.75748   0.79832   0.79832   0.83731   0.83732
 Alpha virt. eigenvalues --    0.85255   0.91941   1.11287   1.12466   1.12466
 Alpha virt. eigenvalues --    1.12856   1.12856   1.15241   1.22590   1.22590
 Alpha virt. eigenvalues --    1.32270   1.54202   1.64443   1.64444   1.72576
 Alpha virt. eigenvalues --    1.91879   1.91879   2.01346   2.13906   2.13907
 Alpha virt. eigenvalues --    2.24455   2.24456   2.29723   2.50423   2.50425
 Alpha virt. eigenvalues --    2.57859   2.63102   2.63102   2.64549   2.74479
 Alpha virt. eigenvalues --    2.74480   3.03447   3.21471   3.85398   3.85400
 Alpha virt. eigenvalues --    4.30725   4.30725   4.35955   4.35956   4.58515
 Alpha virt. eigenvalues --    6.74006   7.71824   7.71826   7.81995   9.87745
 Alpha virt. eigenvalues --   24.04156  26.06659  26.06661  26.25650  48.03918
 Alpha virt. eigenvalues --  215.84969 289.91963 289.91963 290.023261020.81279
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.040632   0.415518   0.415525   0.415498   0.072046  -0.056918
     2  H    0.415518   0.520431  -0.032933  -0.032938  -0.046017  -0.005432
     3  H    0.415525  -0.032933   0.520425  -0.032936  -0.046021  -0.005434
     4  H    0.415498  -0.032938  -0.032936   0.520450  -0.046006  -0.005425
     5  Br   0.072046  -0.046017  -0.046021  -0.046006  35.528496   0.006854
     6  Cl  -0.056918  -0.005432  -0.005434  -0.005425   0.006854  17.838806
 Mulliken charges:
               1
     1  C   -0.302300
     2  H    0.181371
     3  H    0.181375
     4  H    0.181357
     5  Br  -0.469353
     6  Cl  -0.772451
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.241804
     5  Br  -0.469353
     6  Cl  -0.772451
 Electronic spatial extent (au):  <R**2>=           1144.2230
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0020    Y=              0.1066    Z=             -7.4610  Tot=              7.4617
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.7780   YY=            -43.7877   ZZ=            -92.0121
   XY=             -0.0002   XZ=              0.0104   YZ=              0.6896
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             16.0813   YY=             16.0716   ZZ=            -32.1529
   XY=             -0.0002   XZ=              0.0104   YZ=              0.6896
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.0120  YYY=              0.9897  ZZZ=           -187.5842  XYY=             -0.9957
  XXY=              0.1367  XXZ=            -19.6864  XZZ=              0.0302  YZZ=              2.1196
  YYZ=            -19.7086  XYZ=             -0.0147
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -70.2134 YYYY=            -70.5053 ZZZZ=          -1866.4768 XXXY=             -0.0122
 XXXZ=              0.8434 YYYX=              0.0283 YYYZ=             10.1020 ZZZX=              0.2268
 ZZZY=             15.6883 XXYY=            -23.4494 XXZZ=           -256.4684 YYZZ=           -256.5916
 XXYZ=              3.2315 YYXZ=             -0.6538 ZZXY=             -0.0220
 N-N= 1.744696937377D+02 E-N=-7.700984218591D+03  KE= 3.069968212798D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE531\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.652548,-0.00006,0.000212\H,0,-0.842444,-0.898734,-0.559749\H,0,
 -0.84246,0.934229,-0.498045\H,0,-0.842348,-0.03569,1.058489\Br,0,1.598
 398,0.000003,-0.000012\Cl,0,-3.310526,0.000019,-0.000065\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.3901106\RMSD=9.526e-09\Dipole=2.935667
 5,-0.0000622,0.0002198\Quadrupole=-23.9121915,11.9561353,11.9560563,0.
 0002618,-0.0008381,-0.0000072\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 IF MATHEMATICALLY YOU END UP WITH THE INCORRECT ANSWER,
 TRY MULTIPLYING BY THE PAGE NUMBER.
 Job cpu time:       0 days  0 hours  0 minutes 25.3 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 12.9 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:05 2020.
